addons/web_editor/static/src/js/editor/odoo-editor/src/utils/sanitize.js

```diff
@@ -184,6 +189,22 @@ class Sanitize {
                 node.textContent = '\u200B';
             }
 
+            // Ensure the editor tabs allign of the 40px grid
```

Comment:
* allign -> align
* "align of" ? You mean "on" ? If yes then I guess it's "on *a* 40px grid" rather than *the*
* Please add a dot at the end of the sentence (and while you're at it, you can also add one to "Merge identical elements together" above, thanks ^^)

Reply:
:grimacing: